src/components/onboarding.tsx

```diff
@@ -141,7 +159,7 @@ export default function Onboarding() {
 							onSelectPath={ handlePathSelectorClick }
 							error={ error }
 							doesPathContainWordPress={ doesPathContainWordPress }
-							onSubmit={ handleSubmit }
+							onSubmit={ preventEventAndHandleSubmit }
```

Comment:
```suggestion
							onSubmit={ ( event: FormEvent ) => {
										event.preventDefault();
										await onAddSite();
							} }
```

What about this one, remove ` preventEventAndHandleSubmit`, and rename `handleSubmit` to `onAddSite`?

What do you think about that change?

Reply:
Sounds good to me! I added this logic in https://github.com/Automattic/studio/pull/492/commits/424cb682adaa733e9e2cad5056af6aa6553f7f7a